conn.go

```diff
@@ -89,13 +92,22 @@ func createConn(ctx context.Context, nextConn net.Conn, config *Config, isClient
 		loggerFactory = logging.NewDefaultLoggerFactory()
 	}
 
-	logger := loggerFactory.NewLogger("dtls")
+	loggerName := "dtls server"
+	if isClient {
+		loggerName = "dtls client"
+	}
+	logger := loggerFactory.NewLogger(loggerName)
```

Comment:
I don't have a problem letting people pass this in, but probably shouldn't make this change now. People collect logs off of `dtls` right now, so would break them.

Reply:
I'll just revert log related changes. I added this to check trace on CI as it is very hard to see the trace log, but it is not needed in usual applications.